### PR TITLE
Updated script to built chromium tag 84.0.4118.0

### PR DESCRIPTION
**Pull request by ManirajDeivendran**

### CODE REVIEW

File: c/chromium/Chromium_84.0.4118.0_UBI_8.2.sh [hunk 1]
```diff
@@ -4,9 +4,9 @@
 # Package	: chromium
 # Version	: 84.0.4118.0
 # Source repo	: https://chromium.googlesource.com/chromium/src.git
-# Tested on	: UBI 8.2
+# Tested on	: UBI 8.2/UBI 8.4
```
Comment (amitsadaphule) [inline c/chromium/Chromium_84.0.4118.0_UBI_8.2.sh:7]: If we've not tested the updated script on UBI 8.2, let's remove it.

**Review by amitsadaphule — CHANGES_REQUESTED** [review]
Except for the one comment related to base OS, rest looks good to me.

File: c/chromium/Chromium_84.0.4118.0_UBI_8.2.sh (hunk 1 above)
Comment (ManirajDeivendran) [reply c/chromium/Chromium_84.0.4118.0_UBI_8.2.sh]: Removed "UBI 8.2" from comment section. Also renamed the file from Chromium_84.0.4118.0_UBI_8.2.sh to Chromium_84.0.4118.0_UBI.sh

**Review by amitsadaphule — APPROVED** [review]
LGTM

**Pull request merged**